### PR TITLE
Output url

### PR DESCRIPTION
**Pull request by mickmcgrath13**
Update output url

### CODE REVIEW

**Comment by mickmcgrath13** [comment]
tested successfully internally

File: action.yaml [hunk 1]
```diff
@@ -49,6 +49,9 @@ inputs:
   create_vpc:
     description: "Bool, whether a VPC should be created in the action. Default is false"
     default: false
+  infrastructure_only:
+    description: "Bool, does infrastructure (i.e. terraform) but **not** the deployment (i.e. ansible) . Default is true"
+    default: false
```
Comment (LeoDiazL) [inline action.yaml:54]: Wait a minute .... true or false?

File: operations/_scripts/deploy/deploy.sh [hunk 2]
```diff
@@ -53,6 +53,7 @@ docker run --rm --name bitops \
 -e SKIP_DEPLOY_HELM="${SKIP_DEPLOY_HELM}" \
 -e BITOPS_TERRAFORM_COMMAND="${TERRAFORM_COMMAND}" \
 -e TERRAFORM_DESTROY="${TERRAFORM_DESTROY}" \
+-e ANSIBLE_SKIP_DEPLOY="${ANSIBLE_SKIP_DEPLOY}" \
```
Comment (LeoDiazL) [inline operations/_scripts/deploy/deploy.sh:56]: We could make the stack_destroy: true set this to true, so that ansible doesn't run after destroy

**Review by LeoDiazL — COMMENTED** [review]
After adjusting those details, should be good to go

File: operations/_scripts/deploy/deploy.sh (hunk 2 above)
Comment (mickmcgrath13) [reply operations/_scripts/deploy/deploy.sh:56]: good idea!
- https://github.com/bitovi/github-actions-deploy-docker-to-ec2/issues/49
- https://github.com/bitovi/github-actions-deploy-stackstorm/issues/17

File: action.yaml (hunk 1 above)
Comment (mickmcgrath13) [reply action.yaml:54]: 😂 fixed

**Pull request merged**